src/Map.vue

```diff
@@ -1122,6 +1127,16 @@ export default {
      */
     changeMap(newMap) {
       console.log("changeMap:", newMap);
+      this.$router.push(newMap).catch((err) => {
+        // Ignore the vuex err regarding navigating to the page they are already on.
+        if (
+          err.name !== "NavigationDuplicated"
+      && !err.message.includes("Avoided redundant navigation to current location")
```

Comment:
Please rewrite this and use early return pattern with the negated if-clause.

e.g.
```javascript
this.$router.push(newMap).catch((err) => {
  // Ignore the vuex err regarding navigating to the page they are already on.
  if (err.name === "NavigationDuplicated" && err.message && err.message.includes("Avoided redundant navigation to current location")) {
    return;
  }

  // But print any other errors to the console
  console.error(err);
});
```

Also (as I've already done in the suggestion), please make sure the error has a `message` before calling `.includes()` on it.